rslib/src/scheduler/answering/current.rs

```diff
@@ -71,7 +71,7 @@ impl CardStateUpdater {
                     // Decrease reps by 1 to get correct seed for fuzz.
                     // If the fuzz calculation changes, this will break.
                     let last_ivl_with_fuzz = self.learning_ivl_with_fuzz(
-                        get_fuzz_seed_for_id_and_reps(self.card.id, self.card.reps - 1),
+                        get_fuzz_seed_for_id_and_reps(self.card.id, self.card.reps.wrapping_sub(1)),
```

Comment:
```suggestion
                        get_fuzz_seed_for_id_and_reps(self.card.id, self.card.reps.saturating_sub(1)),
```
- https://doc.rust-lang.org/std/intrinsics/fn.wrapping_sub.html
- https://doc.rust-lang.org/std/intrinsics/fn.saturating_sub.html

```rs
fn main() {
    assert_eq!(0_u32.saturating_sub(1), 0);
    assert_eq!(0_u32.wrapping_sub(1), u32::MAX);
}
```

Reply:
I would argue that wrapping sub is preferable here. the context here is to generate a rng seed for a card. with saturating, 0 and 1 both lead to the same seed. and with wrapping 0 and 1 generate different seeds.

this is likely only raising an error because it was built in debug mode for some reason and so is panicing over underflows? so assuming release mode would just let the underflow happen, wrapping maintains the same functionality.

but ultimately its only for generating a seed that nothing important relies on so it doesn't actually matter.

Reply:
Yep, should have precised it but I started with teh saturating one and jake adviced me to switch since we're talking about having a seed value (so no biggies to let it go underflow, even better to keep different seed values for 0 and 1)